docs/UserGuide.md

```diff
@@ -531,6 +528,28 @@ try the command again!
 An example of an error message is shown below.
 ![EditErrMsg.png](images/edit-command/EditErrMsg.png)
 
+### Duplicate detection
+
+_Got an error message while trying to add a company twice? LinkMeIn has got you covered!_
```

Comment:
Makes no sense. Change to a useful use case instead. Something like "Worried about submitting duplicate applications for the same role?" Along those lines.